openwisp_controller/geo/tests/pytest.py

```diff
@@ -107,6 +107,6 @@ async def test_consumer_staff_but_no_change_permission(self):
         await communicator.disconnect()
 
     def test_routing(self):
-        from openwisp_controller.geo.channels.routing import channel_routing
+        from openwisp2.routing import application
```

Comment:
why this change?

you're calling the test project from the test code of the python package, if someone runs the tests from a project which is not called "openwisp2" this test will break. This is not good since we want OpenWISP to be a good base on top of which develop custom applications, which may are named differently and can run these tests as integration tests.

Reply:
From what I understand, the essence of the old test was to check whether the routing is defined properly or not. After refactoring the routes, `openwisp_controller.geo.channels.routing.channel_routing` only exists to offer backward compatibility.

Since we will eventually drop that backward Compatability(say in version 0.10), this test would need to be changed. I thought it is better to clean this up now than leaving it to expire. 

I understand your concern. I don't really see any point in keeping this test. If the routing is not set properly, other tests will fail so the developer will get to know what is wrong. What do you think about dropping this test for good?

Reply:
sounds good, just ensure the test coverage does not decrease too much, thanks!